cmd/plugin/builder/crane/interface.go

```diff
@@ -10,6 +10,9 @@ type CraneWrapper interface {
 	SaveImage(image, pluginTarFilePath string) error
 	// PushImage publish the tar file to remote container registry
 	PushImage(pluginTarFilePath, image string) error
+	// ServeRegistry start an in-memory registry server
+	// returns the port on which server is running, function to shutdown the server, error if any
+	ServeRegistry(port string) (string, func(), error)
```

Comment:
What is the `port` argument used for?   Is it a recommended port value?

Reply:
It is an optional argument if the user wants to run the server on any specific port.

The previous comment was not explaining it that clearly so I have updated the godoc comment for the function.

> // ServeLocalRegistry start an in-memory localhost registry server at provided port
> // If the port is not provided it uses a random available port to start the registry server
> // It returns the port on which the server is running, the function to shutdown the server, error if any
> func ServeLocalRegistry(port string) (string, func(), error) {